README.md

```diff
@@ -16,7 +16,7 @@ To begin, include the `BBCore` and `jQuery` libraries in your html (The latest v
 Then instantiate BBCore:
 
 ```javascript
-var bb = new BBCore({ accessToken: '<your api key>'});
+var bb = new BBCore({ accessToken: '<your access token or api key>', onerror: OnInvalidTokenCallback});
```

Comment:
Was this part existing but just not documented before?

Reply:
Yes